### PR TITLE
[Python][Two-Fer] Clean up, Shorten, and Spell Check Analyzer Comments

### PR DESCRIPTION
**Pull request by BethanyG**
Reviewed the analyzer comments for `two-fer`, and did some editing and spell checking.

### CODE REVIEW

File: analyzer-comments/python/general/no_return.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
 
-A `return` statment is not being used in this function to return the result string, and this code should have failed one or more test cases during testing.
+A `return` statement is not being used in this function to return results.
+ This code should have failed one or more test cases during testing.
 
-Please review your test results.
+Please review the test results, and edit the code accordingly.
 
-If you have not yet run the tests yet for this exercise, please do so now by issuing the command ```run pytest``` inside the `two-fer` exercise directory.
+If you have not yet run the tests for this exercise, please do so now by issuing the command ```run pytest``` inside the problem exercise directory (e.g. `exercism/python/<exercise-name>/`).
```
Comment (angelikatyborska) [inline analyzer-comments/python/general/no_return.md:8]: I'm not sure if this comment makes sense anymore in v3. AFAIK the analyzer doesn't even run if the test runner reported failure.

**Review by iHiD — APPROVED** [review]
(Not checked)

File: analyzer-comments/python/two-fer/wrong_def_arg.md [hunk 2]
```diff
@@ -1 +1 @@
-A value other than 'you' is used as a default argument.
+A value other than 'you' is bing used as a default argument.
```
Comment (angelikatyborska) [inline analyzer-comments/python/two-fer/wrong_def_arg.md:1]: I would personally use DuckDuckGo 😉 😉 😉 (I mean: typo?)

File: analyzer-comments/python/general/no_return.md (hunk 1 above)
Comment (BethanyG) [reply analyzer-comments/python/general/no_return.md:8]: Very good point.  I will delete this.

File: analyzer-comments/python/two-fer/wrong_def_arg.md (hunk 2 above)
Comment (BethanyG) [reply analyzer-comments/python/two-fer/wrong_def_arg.md:1]: 🤣 🤣 🤣   I am constitutionally unable to submit anything without at least **one** typo.  Many thanks for the catch.  😄

File: analyzer-comments/python/general/no_return.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
 # general missing return statement
 
-A `return` statment is not being used in this function to return the result string, and this code should have failed one or more test cases during testing.
+A `return` statement is not being used in this function to return results.
+ This code should have failed one or more test cases during testing.
 
-Please review your test results.
+Please review the test results, and edit the code accordingly.
+<<<<<<< HEAD
+=======
```
Comment (angelikatyborska) [inline analyzer-comments/python/general/no_return.md:8]: 👀

Comment (BethanyG) [reply analyzer-comments/python/general/no_return.md:8]: 😱   _I just can't win..._

**Review by angelikatyborska — APPROVED** [review]

**Pull request merged**